cli/gluster-block.c

```diff
@@ -354,11 +354,16 @@ void
 getCommandString(char **cmd, int argcount, char **options)
 {
   int total_length = 0;
-  int i;
+  int i, ret;
+
   for(i = 1; i < argcount; i++){
     total_length = total_length + strlen(options[i])+1;
   }
-  GB_ALLOC_N(*cmd, (total_length + 1));
+  ret = GB_ALLOC_N(*cmd, (total_length + 1));
+  if (ret) {
```

Comment:
why do we need ret variable, I do not see any use of it.

do you have plans to make this function type as non-void ?